roles/ssh_hardening/README.md

```diff
@@ -41,6 +41,9 @@ Warning: This role disables root-login on the target server! Please make sure yo
 - `ssh_host_key_algorithms`
   - Default: `[]`
   - Description: Host key algorithms that the server offers. If empty the [default list](https://man.openbsd.org/sshd_config#HostKeyAlgorithms) will be used, otherwise overrides the setting with specified list of algorithms.
+- `ssh_client_host_key_algorithms`
+  - Default: `[]`
+  - Description: Specifies the host key algorithms that the client wants to use in order of preference. If empty the [default list](https://man.openbsd.org/ssh_config#HostKeyAlgorithms) will be used, otherwise overrides the setting with specified list of algorithms.
```

Comment:
I think linking to a OpenBSD man source is not preffered here. If possible we should link to RHEL or Debian sources. Especially since the list seems to be version specific. For example the Debian man page (https://manpages.debian.org/stretch/openssh-client/ssh_config.5.en.html#HostKeyAlgorithms) states a very different list of algorithms.

Mybe we should keep this a bit more general here and tell users to look this up in their systems man page?

Reply:
As I walked into exactly that issue for the Host side I agree and will also adapt the description for it.